src/components/AccountDropdown/AccountChildOption.tsx

```diff
@@ -5,7 +5,7 @@ import {
   Stack,
   useColorModeValue,
 } from '@chakra-ui/react'
-import type { AccountId } from '@shapeshiftoss/caip'
+import { type AccountId } from '@shapeshiftoss/caip'
```

Comment:
rever type-only import

Reply:
Reverted in [d831295](https://github.com/shapeshift/web/pull/7663/commits/d83129520fef459ee13122165049b64ef0bc470d)